dags/sentinel-2_gap_filler.py

```diff
@@ -0,0 +1,182 @@
+import csv
+import json
+import sys
+import time
+import boto3
+from datetime import datetime
+from pathlib import Path
+from typing import Dict
+
+
+from airflow import DAG, configuration
+from airflow.hooks.S3_hook import S3Hook
+from airflow.contrib.hooks.aws_sns_hook import AwsSnsHook
+from airflow.contrib.sensors.aws_sqs_sensor import SQSHook
+from airflow.operators.python_operator import PythonOperator
+
+from utils.inventory import s3
+
+default_args = {
+    "owner": "Airflow",
+    "start_date": datetime(2020, 7, 24),
+    "email": ["toktam.ebadi@ga.gov.au"],
+    "email_on_failure": True,
+    "email_on_success": True,
+    "email_on_retry": False,
+    "retries": 0,
+    "s3_report_path": "s3://deafrica-sentinel-2/monthly-status-report/2020-11-11T01:38:02.023140.txt",
+    "report_bucket": "deafrica-sentinel-2",
+    "schedule_interval": "@once",
+    "us_conn_id": "prod-eks-s2-data-transfer",
+    "africa_conn_id": "deafrica-prod-migration",
```

Comment:
Do we need the africa conn?

Reply:
Yes, we do.  Each connection specifies a single region so we need at least two connections.